crates/ef-testing/src/evm_sequencer/evm_state/v0.rs

```diff
@@ -245,12 +245,15 @@ impl Evm for KakarotSequencer {
         transaction: TransactionSigned,
     ) -> TransactionExecutionResult<TransactionExecutionInfo> {
         let evm_address = transaction.recover_signer().ok_or_else(|| {
-            TransactionExecutionError::ValidateTransactionError(
-                EntryPointExecutionError::InvalidExecutionInput {
+            TransactionExecutionError::ValidateTransactionError {
+                error: EntryPointExecutionError::InvalidExecutionInput {
                     input_descriptor: String::from("Signed transaction"),
                     info: "Missing signer in signed transaction".to_string(),
                 },
-            )
+                class_hash: Default::default(),
+                storage_address: Default::default(),
+                selector: Default::default(),
```

Comment:
..Default::default()

Reply:
I don't think we can because `EntryPointExecutionError::InvalidExecutionInput` doesn't derive the default trait